base/challenge-skeleton/README.md

```diff
@@ -1,52 +1,61 @@
-# Quickstart guide to writing a challenge:
+# Quickstart guide to writing a challenge
 
-* A docker image will be built from `challenge/image`. For the simplest challenges, replacing `challenge/image/chal` will be enough.
-* Edit `challenge/image/Dockerfile` to change the commandline or which files should get included.
+The basic steps when preparing a challenge are:
+
+* A Docker image is built from `challenge/image`. For the simplest challenges, replacing `challenge/image/chal` is sufficient.
+* Edit `challenge/image/Dockerfile` to change the commandline or the files you want to include.
 * Try the challenge locally with `make test-docker`.
-* If you have a cluster ready, deploy the challenge with `make start`.
-  * To access it, create a port forward with `make port-forward` and connect via `nc localhost PORT` using the printed port.
+* If you have prepared a cluster, deploy the challenge with `make start`.
```

Comment:
Is it clear what it means to have prepared a cluster? Maybe we can say instead something like "if you have prepared a [cluster](link to how to setup a cluster) already, ..." Or maybe we can just ignore this bullet.

Reply:
adding a link sounds good

Reply:
Cool, maybe we can link to the google-cloud.md walkthrough

Reply:
Added link